pkg/auth/rhsso_authz_handler_test.go

```diff
@@ -93,6 +97,129 @@ var _ = Describe("Authz email domain", func() {
 	}
 })
 
+var _ = Describe("autz_ownership_api", func() {
```

Comment:
Typo, should be `authz_ownership_api`, but also I think it would be more correct to do `Describe("OwnedBy")` here as that's the function under test.